chained/tlsfrag.go

```diff
@@ -0,0 +1,40 @@
+package chained
+
+import (
+	"net"
+	"strconv"
+
+	"github.com/Jigsaw-Code/outline-sdk/transport/tlsfrag"
+	"github.com/getlantern/common/config"
+)
+
+func tlsFragConn(conn net.Conn, pc *config.ProxyConfig) net.Conn {
+	splitHello, index := splitHelloInfo(pc)
+	if splitHello {
+		if _, ok := conn.(*net.TCPConn); !ok {
+			return conn
+		}
+		tlsFragConn, err := tlsfrag.WrapConnFunc(conn.(*net.TCPConn), func(record []byte) (n int) {
+			return index
+		})
+		if err != nil {
```

Comment:
I feel like we should log here so that we have some indication that fragmentation was not used.